src/styles.css

```diff
@@ -36,6 +36,6 @@ header {
     box-shadow: 5px 5px rgb(214, 182, 182);
 }
 
-<link>collapse-content</link> {
-    display: none; /* Hide the content by default */
-  }
\ No newline at end of file
+.collapse-content{
+  color: rgb(119, 119, 119);
+}
```

Comment:
Your original implementation of `collapse-content` wasn't doing anything (and you have used `display:none` "inline" in the html anyway).
So, I stole it to give us a more readable text colour, mostly for my own benefit while testing. We don't need to merge this change.